.github/workflows/.rspec.yml

```diff
@@ -0,0 +1,52 @@
+name: Rspec
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  rspec:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby-version:
+          - 2.6.9
+          - 2.7
+          - 3.0
+          - 3.1
+          - 3.2
```

Comment:
would it be possible to also run against a matrix of rails versions? we can go with combination of rails + ruby depending on min ruby version supported for each rails version. Start with rails 5+, we can drop support for rails 3 and 4.

Reply:
I have made changes in the Gemfile, to install the appropriate `active_record` & `mysql2` versions based on the ruby version. 

Have Ignored Ruby version 2.2.2 - 2.4.10 in the rspec workflow, since support from those versions has ended.